var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -188,35 +205,32 @@ def check_install(self):
             # TODO: Automate these path and library settings
             cc('-c', "-I%s" % join_path(spec.prefix, "include"), "check.c")
             cc('-o', "check", "check.o",
-               "-L%s" % join_path(spec.prefix, "lib"),
-               "-L%s" % join_path(spec.prefix, "lib64"),
```

Comment:
It would probably be easiest to clone your own Spack repository, check out the latest develop branch, and copy your changes to that.